### ISSUE TITLE
how to use tensorboard to record the training loss? thanks

### ISSUE THREAD

**Issue by HejhGitHub** [VainF/DeepLabV3Plus-Pytorch#118]

**Comment by VainF** [comment]
Hi @HejhGitHub. This project was developed four years ago when PyTorch had limited support for Tensorboard. As a result, It only supports Visdom.

**Comment by VainF** [comment]
If you would like to use Tensorboard, just modify these lines using a Tensorboard writer

https://github.com/VainF/DeepLabV3Plus-Pytorch/blob/4e1087de98bc49d55b9239ae92810ef7368660db/main.py#L365

**Comment by HejhGitHub** [comment]
thanks for the reply
